lighthouse-core/test/audits/estimated-input-latency-test.js

```diff
@@ -37,7 +37,7 @@ describe('Performance: estimated-input-latency audit', () => {
     const artifacts = generateArtifactsWithTrace({traceEvents: pwaTrace});
     return Audit.audit(artifacts).then(output => {
       assert.equal(output.debugString, undefined);
-      assert.equal(output.rawValue, 16.4);
+      assert.equal(output.rawValue, 16.2);
```

Comment:
yep, can confirm. the trace for this one now skips the first 15ms and the TTI one skips the first 5ms, adding back this delta yields the old results